cypress/support/pages/dashboard/pageDashboard.js

```diff
@@ -1,32 +1,33 @@
 'use strict'
 
 class Dashboard {
+
   static #welcomeMessage = '[data-testid="welcome-section"]'
   static #dashboardTasksTitle = '[data-testid="dashboard-tasks-title"]'
   static #dashboardTaskCourseName = '[data-testid="task-course-name"]'
   static #dashboardTaskName = '[data-testid="task-name"]'
 
-  seeWelcomeMessage () {
+  seeWelcomeMessage() {
     cy.get(Dashboard.#welcomeMessage)
     cy.contains('Hallo')
   }
 
-  seeAssignedTasks () {
+  seeAssignedTasks() {
     cy.get(Dashboard.#dashboardTasksTitle).eq(0)
     cy.contains('Gestellte Aufgaben')
     cy.get(Dashboard.#dashboardTaskCourseName).eq(0)
-    cy.contains('Natur')
-    cy.get(Dashboard.#dashboardTaskName).eq(0)
-    cy.contains('Test Aufgabe - Copy')
+    cy.contains('Task11')
```

Comment:
Better approach than just commenting out:
  seeAssignedTasks () {
    cy.get(Dashboard.#dashboardTasksTitle).eq(0)
    cy.contains('Gestellte Aufgaben')
    cy.get(Dashboard.#dashboardTaskCourseName).eq(0).contains('Course with subject and tasks')
    cy.get(Dashboard.#dashboardTaskName).eq(0).contains('Task11')
  }

Reply:
Are you planning on doing that?